scripts/airgap/image-list.txt

```diff
@@ -1,4 +1,4 @@
-docker.io/rancher/coredns-coredns:1.6.9
+docker.io/rancher/mirrored-coredns-coredns:1.8.3
```

Comment:
We're not using the mirrored-prefix yet; when we switch we'll switch everything.
Ref: https://github.com/k3s-io/k3s/blob/master/scripts/airgap/image-list.txt#L1